source_modelling/ccldpy.py

```diff
@@ -0,0 +1,1694 @@
+# ccldpy v2.0.0
+# June 12, 2024
+# Tristan E. Buckreis (tristanbuckreis@ucla.edu)
```

Comment:
Can this be used as a package? https://github.com/tristanbuckreis/ccldpy/tree/main

Reply:
I get this with a pip install 

```
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [20 lines of output]
      Traceback (most recent call last):
        File "/var/home/jake/src/.venv/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/var/home/jake/src/.venv/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/var/home/jake/src/.venv/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-build-env-r4aw5311/overlay/lib/python3.12/site-packages/setuptools/build_meta.py", line 332, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=[])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-build-env-r4aw5311/overlay/lib/python3.12/site-packages/setuptools/build_meta.py", line 302, in _get_build_requires
          self.run_setup()
        File "/tmp/pip-build-env-r4aw5311/overlay/lib/python3.12/site-packages/setuptools/build_meta.py", line 516, in run_setup
          super().run_setup(setup_script=setup_script)
        File "/tmp/pip-build-env-r4aw5311/overlay/lib/python3.12/site-packages/setuptools/build_meta.py", line 318, in run_setup
          exec(code, locals())
        File "<string>", line 2, in <module>
      ModuleNotFoundError: No module named 'Cython'
      [end of output]
  ```
Unfortunately, I think this problem can't be remedied by installing the cython dependency prior to installing from git. It would need to be fixed on the ccldpy repo and idk that Tristan is interested in actually maintaining this. I would prefer we rewrote the functionality to use openquake.

Reply:
Hmm, ok. at least there is a newer version. 
If we keep it this way,  perhaps it's a good idea to leave the its github url and a brief note on why we didn't install the package?

Reply:
In principle rewriting ccldpy would not be that hard. It is just generating some random strike and dip values, and comparing a median GMM output.

Reply:
I have added a comment explaining why the ccldpy module is included.